web/generate-geometries.js

```diff
@@ -750,9 +767,14 @@ zoneFeatures = toListOfFeatures(zoneFeaturesInline);
 // Write unsimplified list of geojson, without state merges
 fs.writeFileSync('public/dist/zonegeometries.json', zoneFeatures.map(JSON.stringify).join('\n'));
 
-// Simplify all countries
+// Convert to TopoJSON
 const topojson = require('topojson');
 let topo = topojson.topology(zones);
+
+// merge Florida counties in US-SEC
```

Comment:
maybe add a more explicit description that points to the fact that we do this to avoid having inner lines?

Reply:
agreed, good call, done.